### DISCUSSION TITLE
TerrariumPI/posts/2024/05/11/release-terrariumpi-v-4-9-0/

### DISCUSSION THREAD

**Discussion by giscus** [theyosh/TerrariumPI#921] [Comments]
# TerrariumPI/posts/2024/05/11/release-terrariumpi-v-4-9-0/

Release 4.9.0 - More speed or less waiting

https://theyosh.github.io/TerrariumPI/posts/2024/05/11/release-terrariumpi-v-4-9-0/

**Comment by Konrad84** [comment]
Hi Yosh,

thank you for your developement ! 

I got a problem. I tried to upgrade my TerrariumPI during the Upgrade Process there was some red signs about missing files or something. After Update and reboot i can´t start Terrarium PI. I can get into the Folder but no reaction if i try sudo service terrariumPI start oder sudo ./ run.sh etc. .

Have you an idea what i can do?

**Reply by theyosh** [reply]
Please open an issue. This is not the place for this.

**Comment by Konrad84** [comment]
Oh sorry yes i move the comment to issues
